python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -429,7 +432,13 @@ def _makePanel(self, data, panel, ax, colors, label_font_size=9, legend_font_siz
         if self.panels[panel].referenceValue is not None:
             ax = self._addReferenceLines(ax, panel, panel_range, legend_font_size=legend_font_size)
 
-        if self.panels[panel].statsPanel is None:
+        # check if we should use the default stats panel or if a custom one
+        # has been created.
+        default_stats_panel_bool = self.panels[panel].statsPanel.stat1 is None
+        default_stats_panel_bool &= self.panels[panel].statsPanel.stat2 is None
+        default_stats_panel_bool &= self.panels[panel].statsPanel.stat3 is None
+
```

Comment:
So what happens if you set, say, `stat2`, but not `stat1` or `stat3`?  If it must be all or none, the config settings should be validated.

Reply:
Ok, I think I have setup config validation correctly, so that if must be all or none. 
I also set the task to fail if somehow a incorrectly configured panel makes it to the _makePanel method